frontend/src/container/OnboardingContainer/Modules/APM/GoLang/md-docs/MacOsARM64/Recommended/golang-macosarm64-recommended-runApplication.md

```diff
@@ -6,7 +6,7 @@ Once you are done intrumenting your Go Gin application, you can run it using the
  Run this command inside the `otelcol-contrib` directory that you created in the install Otel Collector step
 
 ```bash
-./otelcol-contrib --config ./config.yaml &> otelcol-output.log & echo "$!" > otel-pid
+./otelcol-contrib --config ./config.yaml &> otelcol-output.log & echo "$\!" > otel-pid
```

Comment:
Is this intentional change? Asking as I am not sure about this change

Reply:
The run statement for MacOs needs to be updated everywhere in the onboarding. 
Will take this up separately. Thankyou for pointing this out!
Have changed it to what it was for now.